apps/core/views.py

```diff
@@ -26,11 +26,94 @@ def get(self, request, *args, **kwargs):
                 ' GROUP BY core_species.name'
             ))
 
+            pixels_by_OmicsUnitType = Pixel.objects.raw((
+                'SELECT core_OmicsUnitType.name AS name, count(*) AS nb,'
+                ' 1 as id'  # this is a hack to overcome Django ORM limitations
+                ' FROM core_pixel'
+                ' JOIN core_omicsunit'
+                '   ON omics_unit_id = core_omicsunit.id'
+                ' JOIN core_OmicsUnitType'
+                '   ON core_omicsunit.type_id = core_OmicsUnitType.id'
+                ' GROUP BY core_OmicsUnitType.name'
+            ))
+
+            pixels_by_OmicsArea = Pixel.objects.raw((
+                'SELECT core_OmicsArea.name AS name, count(*) AS nb,'
+                ' 1 as id'  # this is a hack to overcome Django ORM limitations
+                ' FROM core_pixel'
+                ' JOIN core_PixelSet'
+                '   ON pixel_set_id = core_PixelSet.id'
+                ' JOIN core_Analysis'
+                '   ON core_PixelSet.analysis_id = core_Analysis.id'
+                ' JOIN core_analysis_experiments'
+                '   ON core_Analysis.id = '
+                '   core_analysis_experiments.analysis_id'
+                ' JOIN core_Experiment'
+                '   ON core_analysis_experiments. experiment_id = '
+                '   core_Experiment.id'
+                ' JOIN core_OmicsArea'
+                '   ON core_Experiment.omics_area_id = core_OmicsArea.id'
+                ' where core_OmicsArea.level = 0'
+                ' GROUP BY core_OmicsArea.name'
+            ))
+
+            saved_analysis = Pixel.objects.raw((
+                "Select to_char(saved_at, 'YYYY,MM,DD') AS date,"
```

Comment:
Can you format SQL correctly? At least `SELECT` should be capitalized. I'd say `count` too but it seems I have not done that either, but maybe it would be more consistent to have all SQL keywords uppercased.